gradle/instrumentation.gradle

```diff
@@ -54,3 +51,21 @@ afterEvaluate {
     testImplementation project(':utils:test-utils')
   }
 }
+
+tasks.withType(Test) {
+  useJUnit {
+    excludeCategories 'io.opentelemetry.auto.test.BytecodeTests'
+  }
+}
+
+task integrationTest(type: Test) {
```

Comment:
Looks like we need to add `-x integrationTest` as well to circleci. Though probably better to define a property to handle that with `withType(Test)` globally, maybe `-PskipTests`.

Reply:
Yeah, as usually I forget about CI. Have to review all jobs there

Reply:
Found that `-PskipTests` is already defined :-)
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/gradle/java.gradle#L279